modules/ob-defining-the-container-file.adoc

```diff
@@ -18,14 +18,14 @@ spec:
   source:
     git:
       url: https://github.com/shipwright-io/sample-go
-    contextDir: container-build
+    contextDir: docker-build
   strategy:
     name: buildah
     kind: ClusterBuildStrategy
   paramValues:
-  - name: containerfile
+  - name: dockerfile
     values:
-    - value: Containerfile
+    - value: Dockerfile
```

Comment:
this section should be:
~~~
- name: dockerfile
  value: Dockerfile
~~~
Same syntax has to be followed for all occurrences of paramValues.name for Dockerfile & builder-image.